### PR TITLE
Add sizes options to createSlot

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Will come in handy when we move out of using `data-` attributes, but right now I need it for an upcoming AB test.

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
[Add ability to add new sizes in createSlot](https://trello.com/c/UA9Fp3Te/248-add-ability-to-add-new-sizes-in-createslot)

File: static/src/javascripts-legacy/projects/commercial/modules/dfp/create-slot.js [hunk 1]
```diff
@@ -9,20 +9,19 @@ define([
...
-        mobile: compile(
+        mobile: [
             adSizes.outOfPage,
             adSizes.empty,
             adSizes.mpu,
             adSizes.halfPage,
-            config.page.edition === 'US' ? adSizes.portrait : null,
             adSizes.fluid
-        )
+        ].concat(config.page.edition === 'US' ? adSizes.portrait : [])
```
Comment (katebee) [inline static/src/javascripts-legacy/projects/commercial/modules/dfp/create-slot.js]: Unrelated to PR: why is `adSizes.portrait` unique to the US edition?

Comment (regiskuckaertz) [inline static/src/javascripts-legacy/projects/commercial/modules/dfp/create-slot.js]: Because they are the only ones using it.

**Review by katebee — APPROVED** [review]
:+1:

**Pull request merged**